tool/tctl/common/plugin/plugins_command.go

```diff
@@ -169,6 +176,12 @@ func (p *PluginsCommand) Cleanup(ctx context.Context, args installPluginArgs) er
 	return nil
 }
 
+func (p *PluginsCommand) initUpdateCreds(parent *kingpin.CmdClause) {
+	p.updateCreds.cmd = parent.Command("update-creds", "Updates a plugin's credentials.")
```

Comment:
Sorry, now that I think about it we have `tctl auth rotate` for rotating CAs, would `tctl plugins rotate` make more sense for plugin creds?

Reply:
No worries - either name is fine as long as its consistent.